responders/src/UI/embc-responder/src/app/feature-components/search/essfile-dashboard/ess-file-overview/ess-file-overview.component.html

```diff
@@ -119,44 +119,25 @@
         </div>
         <div class="col-md-9 table-parent">
           <table class="table-details">
-            <tr
-              *ngIf="
-                !essFile?.needsAssessment?.canProvideFood ||
-                essFile?.needsAssessment?.canProvideFood === null
-              "
-            >
-              <th>Food</th>
+            <tr *ngIf="doesIncludeNeed(null)">
```

Comment:
I'd imagine you will remove the *ifs and just bind the array here

Reply:
I can't easily do that because several have different display text than is on the object, either way, somewhere, they have to be processed differently, seperately.  For example, if it's ShelterReferral, the text needs to be just 'Shelter', and if it's 'ShelterAllowance' it needs to be 'Shelter Allowance'